src/platform/web/ui/avatar.js

```diff
@@ -65,6 +70,8 @@ export class AvatarView extends BaseUpdateView {
         this._avatarLetterChanged();
         this._avatarTitleChanged();
         this._root = renderStaticAvatar(this.value, this._size);
+        const image = this._root.firstChild;
```

Comment:
Can we also do this for static avatars (e.g. done with renderStaticAvatar and not binding to the view model)? We'll need to handle both separately though I think, as we need to know in update if the avatar failed to load to add or remove the background color class.